smart-contracts/contracts/RelayPool.sol

```diff
@@ -15,6 +15,8 @@ import "@openzeppelin/contracts/utils/math/Math.sol";
 import {FixedPointMathLib} from "solmate/src/utils/FixedPointMathLib.sol";
 
 struct OriginSettings {
+  uint32 chainId;
+  address bridge;
```

Comment:
Having both bridge and proxy bridge is a bit confusing. It will be good to add a docstring that clarifies all the function params